DigitalLearningSolutions.Web/Helpers/CustomClaimHelper.cs

```diff
@@ -11,7 +11,8 @@ public static class CustomClaimHelper
 
         public static int? GetCandidateId(this ClaimsPrincipal user)
         {
-            return user.GetCustomClaimAsInt(CustomClaimTypes.LearnCandidateId);
+            var id = user.GetCustomClaimAsInt(CustomClaimTypes.LearnCandidateId);
+            return id == 0 ? null : id;
         }
 
         public static int GetCandidateIdKnownNotNull(this ClaimsPrincipal user)

```

Comment:
I wonder if we also need to sort out this method. When I initially created the nullable int version because of the admin users with no delegate ID causing unhandled exceptions, it was purposely left so we didn't have to go and handle the nullable int everywhere. Probably beyond the scope of this ticket but something to think about @stellake

Reply:
Yes, you're right! This is outside the scope and on my TODO! 👍